.travis.yml

```diff
@@ -17,6 +17,7 @@ script:
   - ./gradlew ktlintCheck --no-daemon
   - ./gradlew addKtlintCheckGitPreCommitHook --no-daemon && checkbashisms .git/hooks/pre-commit
   - if [ -n "$TRAVIS_TAG" ]; then ./plugin/gradlew -p ./plugin publishPlugins --no-daemon; fi
+  - if [ -n "$TRAVIS_TAG" ]; then ./plugin/gradlew -p ./plugin githubRelease --no-daemon; fi
```

Comment:
Why not run both tasks in the same build?

Reply:
Updated configuration to use same if condition.